src/libraries/System.IO.Pipelines/src/System.IO.Pipelines.csproj

```diff
@@ -65,4 +65,12 @@ System.IO.Pipelines.PipeReader</PackageDescription>
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="$(SystemThreadingTasksExtensionsVersion)" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">
```

Comment:
Paging @ViktorHofer @ericstj: this PR moves `System.IO.Pipelines` to the shared framework -- are there any other amendements to be made on the infrastructural side of things (including perhaps changes to the NuGet package?)

Reply:
Should the package still be shipped with all the TFMs? (net8.0, net9.0, netstandard2.0 and net462)

Reply:
If so then these changes look right.

Reply:
Agree with @ViktorHofer.  I think you'll probably need to react to this when it reaches Asp.NETCore @wtgodbe.

Reply:
We'll want to remove it from our Shared Framework at that point - thanks for the heads up

Reply:
https://github.com/dotnet/aspnetcore/issues/55342